App/templates/index.html

```diff
@@ -290,9 +286,9 @@ <h1 class="headroom text-center">metro extracts</h1>
       });
     var cities = countries.selectAll(".city").data(function(d){ return d.metros; });
     cities.enter().append("a").attr("class","city");
-    cities.text(function(d){ return d.name.split("_")[0].replace(/-/g," "); })
+    cities.text(function(d){ return d.name; })
       .attr("href",function(d){ 
```

Comment:
@migurski the href property isn't available in the cities.json file 
![screen shot 2016-06-29 at 11 46 07 am](https://cloud.githubusercontent.com/assets/589151/16464443/1fbdbb0c-3def-11e6-8ac5-879bddbb8456.png)